components/apimgt/org.wso2.carbon.apimgt.persistence/src/main/java/org/wso2/carbon/apimgt/persistence/APIPersistence.java

```diff
@@ -144,6 +144,14 @@ void deleteAPIRevision(Organization org, String apiUUID, String revisionUUID, in
      */
     void deleteAPI(Organization org, String apiId) throws APIPersistenceException;
 
+    /**
+     * Delete APIs
+     *
+     * @param org   Organization the API is owned by
+     * @throws APIPersistenceException
+     */
+    void deleteAPIs(Organization org) throws APIPersistenceException;
```

Comment:
if you add deleteAPIs , then you need to pass list of apis. if you want to delete all the apis  then i think name should be deleteAllAPIs()

Reply:
No. we can just delete whole collection calling following way. We don't need the API list
MongoCollection<MongoDBPublisherAPI> collection = MongoDBConnectionUtil.getPublisherCollection(org.getName());
collection.drop();

Reply:
This is renamed right @HiranyaKavishani